crossbar/worker/router.py

```diff
@@ -1016,9 +833,19 @@ def create_resource(self, path_config):
                 else:
                     app = getattr(mod, obj_name)
 
-            # create a Twisted Web WSGI resource from the user's WSGI application object
+            # Create a threadpool for running the WSGI requests in
+            pool = ThreadPool(maxthreads=path_config.get("maxthreads", 20),
+                              minthreads=path_config.get("minthreads", 5),
+                              name="crossbar_wsgi_threadpool")
```

Comment:
Not sure of the semantics either way ;) but 0 is, I think, the equiv. of what you'd have gotten before these changes (if you don't specify minthreads)...